packages/account/src/Components/poi/status/unsupported/card-details/selfie-upload.jsx

```diff
@@ -28,9 +50,13 @@ const SelfieUpload = ({ initial_values, goBack, onConfirm, onFileDrop }) => {
                     return (
                         <Form className={`${ROOT_CLASS}__form`}>
                             <div className={`${ROOT_CLASS}__fields-content`}>
-                                <Text as='h3' size='s' weight='bold' color='prominent'>
-                                    {localize('Upload your selfie')}
-                                </Text>
+                                {is_pa_signup ? (
+                                    selfie_header_for_pa_signup
```

Comment:
@george-usynin-binary Let's render it as a component and also wrap it with `useMemo`.

Reply:
@farzin-deriv sure, will move it to a separate component

Reply:
@george-usynin-binary Yes I meant `React.memo` 😅
Although I see you have moved it outside of the `SelfieUpload` component, So it won’t be created in each rerender and `React.memo` is not necessary anymore.